charms/istio-pilot/lib/charms/istio_pilot/v0/istio_gateway_info_receiver.py

```diff
@@ -0,0 +1,55 @@
+import logging
+from ops.framework import Object
+from ops.model import Application
+
+# Increment this major API version when introducing breaking changes
+LIBAPI = 0
+
+# Increment this PATCH version before using `charmcraft publish-lib` or reset
+# to 0 if you are raising the major API version
+LIBPATCH = 1
+
+
+DEFAULT_RELATION_INTERFACE = "gateway"
+
+
+class GatewayConsumer(Object):
```

Comment:
Although I agree that Consumer is a more intuitive name we should probably stick with Requirer so that we use the same terminology as in the juju metadata.